### PR TITLE
Tune fixed GAS limits

### LINKED ISSUE #2037: Tune fixed GAS limits
See neo-project/neo#2493 and neo-project/neo-modules#604.

### PR DESCRIPTION
**Pull request by fyrchik**
Close #2037 .

### CODE REVIEW

**Review by roman-khimov — CHANGES_REQUESTED** [review]
Notice that headers have `3_00000000L` limit.

File: pkg/core/blockchain.go [hunk 1]
```diff
@@ -50,7 +50,7 @@ const (
 	defaultMaxBlockSystemFee               = 900000000000
 	defaultMaxTraceableBlocks              = 2102400 // 1 year of 15s blocks
 	defaultMaxTransactionsPerBlock         = 512
-	verificationGasLimit                   = 100000000 // 1 GAS
+	verificationGasLimit                   = 1_50000000 // 1.5 GAS
```
Comment (AnnaShaleva) [inline pkg/core/blockchain.go:53]: We also need to increase GAS for headers verification https://github.com/neo-project/neo/pull/2493/files#diff-b79f1f745f19a6d42055888b9cf96063394726ba2603a25af58547313f6d7518R222-R223

**Review by AnnaShaleva — CHANGES_REQUESTED** [review]

File: pkg/core/blockchain.go (hunk 1 above)
Comment (AnnaShaleva) [reply pkg/core/blockchain.go:53]: Oh, I see it's already mentioned in https://github.com/nspcc-dev/neo-go/pull/2055#pullrequestreview-704039053

**Comment by AnnaShaleva** [comment]
> While we decode `CMDHeaders` payload, we don't use it in any way. It is added in #2019 though.

So I can fix it in #2019 then.

**Comment by roman-khimov** [comment]
I'd rather have it there (it's quite a simple change that technically has nothing to do with the essence of #2019). #2019 might be postponed after 0.96.0.

**Comment by roman-khimov** [comment]
@fyrchik?

**Comment by fyrchik** [comment]
Sorry, forgot to push changes.

**Review by roman-khimov — APPROVED** [review]

**Review by AnnaShaleva — APPROVED** [review]

**Pull request merged**